package/samplers/hebo/sampler.py

```diff
@@ -1,12 +1,18 @@
 from __future__ import annotations
 
+from typing import Optional
+from typing import Sequence
```

Comment:
We should not use typing.Sequence. We should use collections.abc.Sequence.

Reply:
Fixed.